src/compas_fab/ghpython/components/Cf_VisualizeTrajectory/metadata.json

```
@@ -0,0 +1,61 @@
+{
+    "name": "Trajectory Visualize",
+    "nickname": "Trajectory Visualize",
+    "category": "COMPAS FAB",
+    "subcategory": "Display",
+    "description": "Visualizes the trajectory.",
+    "exposure": 2,
+
+    "ghpython": {
+        "isAdvancedMode": true,
+        "inputParameters": [
+            {
+                "name": "robot",
+                "description": "The robot."
+            },
+            {
+                "name": "group",
+                "description": "The planning group for which this trajectory was planned.",
+                "typeHintID": "str"
+            },
+            {
+                "name": "trajectory",
+                "description": "The calculated trajectory."
+            }
+        ],
+        "outputParameters": [
+            {
+                "name": "start_configuration",
+                "description": "The start configuration of the trajectory."
+            },
+            {
+                "name": "configurations",
+                "description": "The full configurations along the trajectory."
+            },
+            {
+                "name": "fraction",
+                "description": "Indicates the percentage of requested trajectory that was calculated, e.g. 1 means the full trajectory was found."
```

"fraction", I think, is more accurate.
```suggestion
                "description": "Indicates the fraction of requested trajectory that was calculated, e.g. 1 means the full trajectory was found."
```